cmd/cluster/encryption/encryption.go

```diff
@@ -70,6 +70,50 @@ var listCmk = &cobra.Command{
 	},
 }
 
+var updateCmkState = &cobra.Command{
+	Use:   "update-state",
+	Short: "Update Encryption at Rest (EaR) state for a cluster",
+	Long:  "Update Encryption at Rest (EaR) state for a cluster",
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf("Could not initiate api client: %s", err.Error())
```

Comment:
This might be confusing to the user, we need to make this as a generic error @cdavid

Reply:
This error is just everywhere and it s indeed an issue with the api client, not sure what you mean by generic ? Can you explain a bit more ?

Reply:
This error does not provide any guidance to the user. Can we simply report internal error? or unable to connect to Yugabyte Managed Service

Reply:
Interesting, I think  " unable to connect to Yugabyte Managed Service"  is too vague, at least API do return some useful information like 401 auth... or other. But no strong opinion on that.

We can change `Could not initiate api client` but we need to keep the `err.Error())` otherwise will too generic and difficult to support.

Reply:
Agreed

Reply:
Please file a JIRA for this